raiden/utils/upgrades.py

```diff
@@ -166,6 +167,18 @@ def run(self) -> None:
                 f"older."
             )
 
+        if RAIDEN_DB_VERSION >= 27 and file_version <= 26 and file_version > 1:
+            msg = (
+                f"Your Raiden database is version {file_version} and there is no compatible "
+                f"migration to version {RAIDEN_DB_VERSION} available.\n"
+                "You need to either start a new Raiden node with a different account, or "
+                "close and settle all channels, and start over with a fresh database.\n\n"
+                "If you are on **mainnet** and affected by this, please create an issue at "
+                "https://github.com/raiden-network/raiden/issues/new?title=Mainnet%20Migration"
+            )
+            log.warning(msg)
+            sys.exit(msg)
```

Comment:
TIL that one can pass a string to `sys.exit()`...

Reply:
me too :)